strongbox-cron/strongbox-cron-rest/src/test/java/org/carlspring/strongbox/cron/controller/CronTaskConfigurationControllerTest.java

```diff
@@ -48,22 +51,19 @@ public void init()
     }
 
     @Test
-    public void downloadRemoteMavenIndexCronJobShouldHaveWorkingPreCreateCallback()
+    public void putDownloadRemoteMavenIndexCronJob()
             throws Exception
     {
-        Assume.assumeTrue(context.containsBean("repositoryIndexManager"));
+        Assert.assertTrue(context.containsBean("repositoryIndexManager"));
```

Comment:
Could you please use static imports for all `asserts*()` methods? @ptirador has already cleaned these up in #871 for the whole code base. Thanks! :)